config/motd.txt

```diff
@@ -2,16 +2,18 @@
 
 <h1>Welcome to Bad Deathclaw!</h1>
 
-<b>THIS IS AN 18+ SERVER, LEAVE NOW IF YOU ARE NOT OLD ENOUGH (also leave if you like to act under 18)</b>
+<b>This is an 18+ server, do not play if you are underage.</b>
 
 <i>
-This is a lax medium RP server meaning you're free to do things such as kidnap and rape but please remember you're not the only person trying to have fun. Don't kill for no reason, do not kill on sight, don't be a huge dick and <b>read the rules!</b>.
+This is a light to medium roleplay server, freedom is given to ensure the continuity of the round, however interpretation of this freedom is at the discretion of the admins. Don't be a dick, and <b>read the rules</b> before you play.
```

Comment:
"Don't be a dick, and <b>read the rules</b> before you play."

Reply:
Edit it down to just those words, please. The rest of it is unnecessary or covered in the rules.